airbyte-webapp/src/locales/en.json

```diff
@@ -281,6 +281,13 @@
   "sources.emptySchema": "Schema is empty",
   "sources.noSources": "Source list is empty",
   "sources.sourceSettings": "Source Settings",
+  "sources.documentationPanel.tabs.setupGuide": "Setup Guide",
+  "sources.documentationPanel.tabs.schema": "Schema",
+  "sources.documentationPanel.tabs.erd": "ERD (Entities Relationship Diagram)",
+  "sources.request.button.schema": "Request schema for this source",
+  "sources.request.button.erd": "Request ERD for this source",
+  "sources.request.prioritize": "We're still building this feature - if you'd like us to prioritize this source.",
```

Comment:
I think the wording here is a bit off (grammar wise), but will refer to @nataliekwong as a native speaker.

"We're still building this feature - if you'd like us to prioritize this source." feels like language wise something is off?

I'd suggest something like: "We're still building this feature for you. Let us know if you're interested in {ERD diagrams|Schemas} for this source."?

Reply:
cc @nataliekwong Would appreciate your input on this still, because we really don't want to trust my lousy English skills :D

Reply:
> I think it's worth passing by the founders as to whether we should experiment in OSS before getting signals from Cloud. I plan to write up a short doc on this by Thursday.

I chatted with Jean briefly about the open question of whether we put experiments in OSS. His feedback was that we should not experiment in OSS, but instead pass our learnings into OSS once we’ve established something works in Cloud.

In this specific example, if we see that users do readily request example data or ERDs, then that feature could move to OSS. But without validating that users would find that useful, we shouldn’t push it to OSS. @letiescanciano @timroes

Reply:
Suggestion: We're still building this feature for you. Let us know if you're interested in a {Entity Relationship Diagram (ERD) | schema} for this source.

I wonder if people actually know what these are.  I was thinking we could actually send them to an example of what one looks like, but also don't want them to leave their existing flow.  Is it possible to show an example, but greyed out / shrouded in the background so that it's clear it's not to be actually used?